### PR TITLE
Natural RTL/LTR support

### PR DESCRIPTION
**Pull request by danielgindi**

### CODE REVIEW

File: TagListView/TagListView.swift [hunk 1]
```diff
@@ -230,30 +232,58 @@ open class TagListView: UIView {
             $0.removeFromSuperview()
         }
         rowViews.removeAll(keepingCapacity: true)
+
+        var isRtl: Bool = false
```
Comment (Cee) [inline TagListView/TagListView.swift]: Should be good to move this property out.

File: TagListView/TagListView.swift [hunk 2]
```diff
@@ -126,8 +126,10 @@ open class TagListView: UIView {
         case left
         case center
         case right
+        case start
+        case end
```
Comment (Cee) [inline TagListView/TagListView.swift]: I would not prefer to add these two cases here. It's ambiguous when you say "left"/"start" in RTL languages.

File: TagListView/TagListView.swift (hunk 1 above)
Comment (danielgindi) [reply TagListView/TagListView.swift]: Nope :-) It's not a property, it's a local variable, deduce from the UIView's properties.
iOS already has mechanisms in place to set LTR/RTL, with semanticAttribute and stuff.
Just the method to get the proper final decision on RTL changes between iOS versions, so there's an "@if" there

File: TagListView/TagListView.swift (hunk 2 above)
Comment (danielgindi) [reply TagListView/TagListView.swift]: > I would not prefer to add these two cases here. It's ambiguous when you say "left"/"start" in RTL languages.

That's incorrect.
Apple usually uses "left/right/center/leading/trailing", google and all the rest are using "left/right/center/start/end".
It;s basically like the "natural" option in UILabel alignment, but allowing both sides, automatically based on direction.

Comment (xhacker) [reply TagListView/TagListView.swift]: Can you use `leading` and `trailing`?

File: TagListView/TagListView.swift [hunk 3]
```diff
@@ -230,30 +232,58 @@ open class TagListView: UIView {
             $0.removeFromSuperview()
         }
         rowViews.removeAll(keepingCapacity: true)
+
+        var isRtl: Bool = false
+        
+        if #available(iOS 10.0, tvOS 10.0, *) {
+            isRtl = effectiveUserInterfaceLayoutDirection == .rightToLeft
+        } else if #available(iOS 9.0, *) {
```
Comment (xhacker) [inline TagListView/TagListView.swift]: Can you put `else if` to a new line, to match the current style of the project?

**Comment by xhacker** [comment]
Looks good overall! Please also add some documentation in `README.md`.

File: TagListView/TagListView.swift (hunk 2 above)
Comment (danielgindi) [reply TagListView/TagListView.swift]: Done!

File: TagListView/TagListView.swift (hunk 3 above)
Comment (danielgindi) [reply TagListView/TagListView.swift]: Done!

**Comment by danielgindi** [comment]
> Looks good overall! Please also add some documentation in `README.md`.

Done

**Comment by danielgindi** [comment]
@csvenja Any plans for a release?

**Comment by summonerriftofficial** [comment]
@danielgindi will in my case I guess you need to make semantic for to right, it works for me

**Pull request merged**